specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/stable/2020-10-01/azurestackhci.json

```diff
@@ -561,7 +561,10 @@
           "type": "array",
           "items": {
             "$ref": "#/definitions/OperationDetail"
-          }
+          },
+          "x-ms-identifiers": [
```

Comment:
Why is this been added only for these two API versions? (2020-10-01 and 2022-01-01)

Reply:
@priyjain358  We only add the xms identifiers to the most recent api versions (stable) and the places pointed by running this command : `autorest --validation --azure-validator --openapi-type=arm --use=@microsoft.azure/classic-openapi-validator@1.1.13 --use=@microsoft.azure/openapi-validator@1.12.1 "specification/azurestackhci/resource-manager/readme.md"
`  ; We add them to the versions that we are using (Change Analysis is an extension RP) just give an example to the RP teams on how to add them. If you want to propagate to other versions, feel free to do so.